ci/infra/aws/README.md

```diff
@@ -71,16 +71,16 @@ Once the infrastructure has been created, you can obtain the details with
 
 ```console
 $ terraform output
-control_plane.private_dns = [
-    ip-10-1-1-55.eu-central-1.compute.internal
-]
-control_plane.public_ip = [
-    3.121.219.168
-]
+control_plane.private_dns = {
+    i-1234567890abcdef0 = ip-10-1-1-55.eu-central-1.compute.internal
```

Comment:
Does not know what `i-1234567890abcdef0` means? IP or FQDN or ?

Reply:
its an aws instance id, only part of an aws instance that is sure to be there (host, host_id, tags are all optional)

Reply:
:ok: